public/app/plugins/panel/logs/LogsPanel.tsx

```diff
@@ -350,7 +348,13 @@ export const LogsPanel = ({
         scrollTop={scrollTop}
         scrollRefCallback={(scrollElement) => setScrollElement(scrollElement)}
       >
-        <div className={style.container} ref={logsContainerRef}>
+        <div
+          onMouseLeave={() => {
```

Comment:
Can you `useCallback` this one? Not a big deal, but consistent with what we've done with the Panel.

Reply:
Done!